src/Fabulous.Avalonia/Widgets/Controls/TimePicker.fs

```diff
@@ -0,0 +1,61 @@
+namespace Fabulous.Avalonia
+
+open System
+open System.Runtime.CompilerServices
+open Avalonia.Controls
+open Fabulous
+
+type IFabTimePicker =
+    inherit IFabTemplatedControl
+
+module TimePicker =
+    let WidgetKey = Widgets.register<TimePicker> ()
+
+    let ClockIdentifier =
+        Attributes.defineAvaloniaPropertyWithEquality TimePicker.ClockIdentifierProperty
+
+    let MinuteIncrement =
+        Attributes.defineAvaloniaPropertyWithEquality TimePicker.MinuteIncrementProperty
+
+    let SelectedTime =
```

Comment:
As this will always initialize by the constructor widget. We can safely ignore the converters.  We can use 
```fsharp
let SelectedTime =
        Attributes.defineAvaloniaPropertyWithEquality TimePicker.SelectedTimeProperty
```

Reply:
Ah, that makes sense. I've just committed this change, along with the analogous change to DatePicker's SelectedDate property